tests/math/test_matrix_manipulation.py

```diff
@@ -23,6 +23,17 @@
 import pennylane as qml
 from pennylane import numpy as pnp
 
+# Import interfaces
+autograd = pytest.importorskip("autograd")
+tf = pytest.importorskip("tensorflow")
+torch = pytest.importorskip("torch")
+jax = pytest.importorskip("jax")
+jnp = pytest.importorskip("jax.numpy")
+
+# Define a list of dtypes to test
+dtypes = ["complex64", "complex128"]
+array_funcs = [lambda x: x, np.array, pnp.array, jnp.array, torch.tensor, tf.Variable, tf.constant]
```

Comment:
I see how this is convenient but the way our tests with all four interfaces are structured is that they should be in separate tests and individually marked with `@pytest.mark.jax` etc.
One of the reasons being that the test suite is running separately jax, torch and tf test runners in CI. I am actually surprised this worked 🤔

Reply:
@timmysilv made me aware of this neat trick with which you can still parametrize your tests and dont have to rewrite them for all four interfaces :)
Use `pytest.param` like here https://github.com/PennyLaneAI/pennylane/blob/d5e3cca6a368a4ef2fccc73dfa6e5ab1f0cbb202/tests/devices/qubit/test_apply_operation.py#L32-L38
and then parametrize using `qml.math.asarray(x, like=param)` like here https://github.com/PennyLaneAI/pennylane/blob/d5e3cca6a368a4ef2fccc73dfa6e5ab1f0cbb202/tests/devices/qubit/test_apply_operation.py#L83

Reply:
this is certainly handy! Thanks for bringing this to my attention! Did add this in there